src/miniscript/astelem.rs

```diff
@@ -460,9 +469,9 @@ impl_from_tree!(
             ("pk_k", 1) => {
                 expression::terminal(&top.args[0], |x| Pk::from_str(x).map(Terminal::PkK))
             }
-            ("pk_h", 1) => {
-                expression::terminal(&top.args[0], |x| Pk::Hash::from_str(x).map(Terminal::PkH))
-            }
+            ("pk_h", 1) => expression::terminal(&top.args[0], |x| {
+                Pk::Hash::from_str(x).map(Terminal::RawPkH)
```

Comment:
We should never parse Pk::Hash. We should only parse Pk and store it in PkH. `RawPkH` should never be created as it is unsupported.

Reply:
Right, will address this shortly!

Reply:
Right, will address this shortly!